### PR TITLE
Test deploy to netlify

### PR DESCRIPTION
**Pull request by CodingCarlos**
Let try to have a test deploy to Netlify, so we can better test and check all the changes before moving to production

### CODE REVIEW

**Comment by CodingCarlos** [comment]
# TL;DR:
**We're not using Netlify to deploy this. We're going to Vercel. Check #33 PR**

## What happened here?
We are not going to use Netlify to deploy this repo, so I'm closing this issue.

## Why?
Because netlify only allows Nuxt to be deployed in static mode, or in Client Render Only. 

As the production deploy is Server Side Rendered, I don't want to have different settings in the preview or test deployments. So, netlify is not an option.

## What do we do, so?
There's plenty of options, such as heroku or vercel, to name some. I've done a quick research, and I think vercel might be a cool option, since it has integrated nuxt settings. That means, that we can have (kind of) the same settings in the auto-deploys than in production deployments, so that's amazing.

### How do we move to Vercel?  
Easy. Check out the #33 PR. Everything is there.
